File: Java/Iniciante/1007 - Diferença..java
/*
* Autor: Pierre Vieira
* Data de submissão: 19/01/2020 17:10:54
*/
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int a, b, c, d;
        a = teclado.nextInt();
        b = teclado.nextInt();
        c = teclado.nextInt();
        d = teclado.nextInt();
        System.out.printf("DIFERENCA = %d\n", (a*b - c*d));
    }
}

File: C#/Iniciante/1007 - Diferença.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 12:14:40
*/
using System;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int a, b, c, d, diferenca;
            a = int.Parse(Console.ReadLine());
            b = int.Parse(Console.ReadLine());
            c = int.Parse(Console.ReadLine());
            d = int.Parse(Console.ReadLine());
            diferenca = a * b - c * d;
            Console.WriteLine("DIFERENCA = "+diferenca);
        }
    }
}
